.travis.yml

```diff
@@ -1,14 +1,26 @@
 ---
 sudo: required
 services: docker
-language: generic
+language: python
 dist: xenial
 
+python:
+  - "3.6"
+
 branches:
   only:
   - master
   - travis
 
+before_script:
+  - sudo apt-get update
+  - sudo apt-get install google-chrome-stable=73.0.3683.86-1 -y
+  - wget https://chromedriver.storage.googleapis.com/73.0.3683.68/chromedriver_linux64.zip
```

Comment:
Any specific reason you needed to move to chrome 73 ? :smile:

Reply:
I am actually not moving backward to chrome 73, because it is newest available version of `google-chrome-stable` on this travis. I only added check that that version is installed, because it will work with `chromedriver` :smile: